modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/copy_template_folder_command.rb

```diff
@@ -62,9 +62,7 @@ def call(source_location:, destination_name:)
           def handle_response(response)
             case response
             in { status: 202 }
-              id = extract_id_from_url(response.headers[:location])
-
-              ServiceResult.success(result: { id:, url: response.headers[:location] })
+              ServiceResult.success(result: { id: nil, url: response.headers[:location] })
```

Comment:
🟡 can we skip the property `id` if it is always `nil` when using the copy command of OneDrive?

Reply:
Yeah, that was the idea. Be explicit about the nil id. =)